app/controllers/admin_controller.rb

```diff
@@ -7,32 +7,42 @@ def allow_only_admins
     # This shouldn't be needed becuse we also check for admin-ness in the
     # routes. Still doing this just to be safe.
     authenticate_user!
-    if not current_user.admin?
-      raise ActionController::RoutingError.new('Not Found')
-    end
+
+    fail ActionController::RoutingError 'Not Found' unless current_user.admin?
```

Comment:
Tests are throwing and error here:

`Message: <"undefined method 'RoutingError' for ActionController:Module">`

without the `new()` method, ruby think that `RoutingError` is a method and not a class